source/development/feature-flags.html.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Feature flags in GOV.UK Frontend
+weight: 1
+last_reviewed_on: 2023-11-28
+review_in: 6 months
+---
+# Using feature flags in GOV.UK Frontend
+
+Feature flags are a way for us to do a 'soft' release of large scale or experimental work by allowing users to opt in our out of a new, breaking change or feature. Read this guidance to decide whether a feature flag is a useful addition to your project.
```

Comment:
```suggestion
Feature flags are a way for us to do a 'soft' release of large scale or experimental work. They allow users to opt in or out of changes that are new, breaking or a feature. Read this guidance to decide whether a feature flag is a useful addition to your project.
```

Reply:
This slightly changes the meaning of the end of the first sentence, now the second sentence with your suggestion. I was trying to say that a user can now opt out of a feature or change that is new or breaking. Maybe something like...

>They allow users to opt in our out of new or breaking changes.

Thonks?

Reply:
Yup, ok cool. But should it be 'or' rather than 'our'?